modules/core/src/main/java/org/apache/ignite/internal/processors/cache/GridCacheProcessor.java

```diff
@@ -2626,7 +2627,11 @@ private void stopGateway(DynamicCacheChangeRequest req) {
      * @param clearDbObjects If {@code false} DB objects don't removed (used for cache.close() on client node).
      * @return Stopped cache context.
      */
-    public GridCacheContext<?, ?> prepareCacheStop(String cacheName, boolean destroy, boolean clearDbObjects) {
+    @Nullable public GridCacheContext<?, ?> prepareCacheStop(
```

Comment:
no one use return result from this func, may it would be be more clear to change implementation to void ?